pfioh/Auth.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
```

Comment:
Why?

Reply:
This is just a python convention. It's not required, but it's very common. Most of the classes in the pfioh repo have shebang lines.

Reply:
I guess it doesnt really belong since this will never be run as an executable though